src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -90,16 +90,16 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
 
                 IReadOnlyList<IREvaluationResultInfo> infoList = null;
 
-                async Task FillInfoList() {
+                Task.Run(async () => {
```

Comment:
What was the issue?
`TryEvaluateAndDescribeAsync` is always executed on background thread, so the reason for `Task.Run` can be if current method should proceed while evaluation is running. But since we're synchronously waiting for the task, I don't understand the purpose.

Reply:
Performance regression per Hong. Type `mtcars$` - 2 sec delay, then nothing.